platform/contact-center/training-video-process.md

```diff
@@ -1,9 +1,17 @@
-### Contact Center Training Video Process
+# Contact Center Training Video Process
 
-## Summary
-Training videos are the video version of the product guide and are created by VFS teams for contact center use.  The contact center training videos can easily be created by recording a Slack, Zoom, Microsoft Teams, or Skype call.  
+**Overview:**: How to create Product Videos for the Contact Center Review in the Build Stage of the Collaboration Cycle.
 
-During the video the team will walk though the primary flows of the product, using layman terms whenever possible. 
+**Audience**: VFS teams building on VA.gov
+
+## Process
+
+Training videos are the video version of the product guide and are created by VFS teams for contact centers to help Veterans troubleshoot VA.gov.  
+
+When your team is at the [Prepare for Contact Center Review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/request-contact-center-review.md#prepare-for-contact-center-review) stage, reach out to the Contact Center POCs on #vsp-contact-center-support Slack channel to set up a video meeting. The contact center training videos will be created by recording the Zoom call so that we can capture the closed captioning.  
+
+During the video, your team will walk though the primary flows of the product, using layman terms whenever possible. 
```

Comment:
Let's provide a bit more detail here by saying that demo videos will vary in length depending on the complexity of the product. Demo videos should cover how to navigate to the product, user access (i.e. demonstrating if you need to sign in) and then the functionality of the product. You do not need to write a script, but for more complex products you might want to write down what you will show and in what order to make sure you capture everything for the demo.